packages/lambda/src/cli/commands/render/render.ts

```diff
@@ -66,6 +66,7 @@ export const renderCommand = async (args: string[]) => {
 		quality,
 		scale,
 		everyNthFrame,
+		// loop=0 results in infinite
```

Comment:
`loop=0` should result in infinite loop.

- -1 no loop (plays once)
- 0 infinite loop (default)
- 1 loop once (plays 2 times)
- 2 loop twice (plays 3 times) 

[source](https://superuser.com/a/1607134/1665061)

Reply:
Yes we can definately do that, I'll make the changes.

Reply:
You are of course right @Iamshankhadeep 🙈